lib/dmv_factory.rb

```diff
@@ -0,0 +1,24 @@
+class DmvFactory 
+  attr_reader :office_list
+  def initialize
+    @office = nil
+  end
+
+  def create_offices_helper(api_data)
+    @office_list = api_data.map do |office|
+     office_data = { 
+      name: "#{office[:dmv_id]} #{office[:dmv_office]}", 
+      address: "#{office[:address_li]} #{office[:address__1]} 
+                #{office[:city]} #{office[:state]} #{office[:zip]}", 
+      phone: office[:phone], services: office[:services_p] } 
+    end
+  end
```

Comment:
I think this method could use some additional formatting to make it easier to read.

```ruby
def create_offices_helper(api_data)
  @office_list = api_data.map do |office|
    office_data = { 
      name: "#{office[:dmv_id]} #{office[:dmv_office]}", 
      address: "#{office[:address_li]} #{office[:address__1]} #{office[:city]} #{office[:state]} #{office[:zip]}", 
      phone: office[:phone], 
      services: office[:services_p] 
    } 
  end
end
```

You also do not need to be grabbing `services` data from the hash since a new Facility object will start with no services.